x-pack/legacy/plugins/siem/public/components/ml_popover/translations.ts

```diff
@@ -28,7 +28,7 @@ export const UPGRADE_DESCRIPTION = i18n.translate(
   'xpack.siem.components.mlPopup.upgradeDescription',
   {
     defaultMessage:
-      'In order to access SIEM’s anomaly detection features, you must be subscribed to an Elastic Platinum license. With it, you’ll have the ability to run Machine Learning jobs to view anomalous events throughout SIEM.',
+      'To access SIEM’s anomaly detection features, you must update your license to Platinum or start a free 30-day trial. You can then run Machine Learning jobs and view anomalies.',
```

Comment:
Suggested changes for mentioning cloud:

To access SIEM’s anomaly detection features, you must update your license to Platinum, start a free 30-day trial,  or spin up a deployment on AWS, GCP, or Azure at https://www.elastic.co/cloud/.  You can then run Machine Learning jobs and view anomalies.

Reply:
Updated the text